packages/snaps-sdk/src/types/methods/end-trace.ts

```diff
@@ -0,0 +1,35 @@
+import type { TraceName } from './start-trace';
+
+/**
+ * A request to end a pending trace.
+ */
+export type EndTraceRequest = {
+  /**
+   * The unique identifier of the trace.
+   * Defaults to 'default' if not provided.
```

Comment:
Wait really?

Reply:
Apparently. This is copied from the extension types.